api-common/src/main/kotlin/io/qalipsis/api/meters/CampaignMeterRegistry.kt

```diff
@@ -68,6 +116,18 @@ interface CampaignMeterRegistry {
         number: T,
```

Comment:
I think that it would be better not to provide the number as argument. The implementation of Gauge should take care of it. This will make it then possible to share across several factories.